tests/test-admin-deploy-etcmerge-cornercases.sh

```diff
@@ -66,6 +78,12 @@ newetc=${newroot}/etc
 assert_file_has_content ${newroot}/usr/etc/NetworkManager/nm.conf "a default daemon file"
 assert_file_has_content ${newetc}/NetworkManager/nm.conf "a modified config file"
 
+if test -e "${newetc}"/sock-to-ignore; then
+  fatal "Should not have copied socket!"
+fi
+kill $(jobs -p)
```

Comment:
Maybe let's just capture the PID of the netcat when you spawn it (`$!`) and use that here and in the trap?

Reply:
Or, maybe it's simpler to just test with a FIFO instead for this test?

Reply:
Yeah, done - we have socket coverage with the other test.